src/parsing/parsers/generic/jison/alter/alter_common.test.ts

```diff
@@ -11,4 +11,16 @@ test('should suggest ALTER', () => {
 
     const suggestion: KeywordSuggestion = { value: 'ALTER', weight: -1 };
     expect(parseResult.suggestKeywords).toContainEqual(suggestion);
+})
+
+test('should suggest ALTER objects', () => {
+    const parseResult = parseGenericSql('ALTER ', '');
+
+    expect(parseResult.errors).toBeUndefined();
+
+    const suggestions: KeywordSuggestion[] = [
+        { value: 'TABLE', weight: -1 },
+        { value: 'VIEW', weight: -1 },
+    ];
+    expect(parseResult.suggestKeywords).toEqual(suggestions)
```

Comment:
I've decided to move those tests closer to their definition. Even though it can be separated to `alter_table`, `alter_view` files. But the actual code that suggests it is `alter_common`, so it's better this way.